lab_3_keywords_textrank/main.py

```diff
@@ -734,7 +868,28 @@ def train(self) -> int:
             int:
                 0 if importance scores were calculated successfully, otherwise -1
         """
-        pass
+        phrases = extract_phrases(self._text)
+        if not phrases:
+            return -1
+        candidate_keyword_phrases = extract_candidate_keyword_phrases(phrases, list(self._stop_words))
+
+        if not candidate_keyword_phrases:
+            return -1
+        content_word_frequencies = calculate_frequencies_for_content_words(candidate_keyword_phrases)
+
+        if not (candidate_keyword_phrases and content_word_frequencies):
+            return -1
+        list_content_word_frequincies = list(content_word_frequencies)
+        word_degrees = calculate_word_degrees(candidate_keyword_phrases, list_content_word_frequincies)
+
+        if not (word_degrees and list_content_word_frequincies):
```

Comment:
correct me if i'm wrong, but `list_content_word_frequincies`  can only be equal to False if corrupt arguments were passed to `calculate_frequencies_for_content_words` which you've checked
so, this is redundant?